plugins/stats_table_procedure.cc

```diff
@@ -110,18 +122,18 @@ save(const std::string & filename) const
 void StatsTable::
 serialize(ML::DB::Store_Writer & store) const
 {
-    int version = 1;
+    int version = 2;
     store << version << colName << outcome_names << counts << zeroCounts;
 }
 
 void StatsTable::
 reconstitute(ML::DB::Store_Reader & store)
 {
     int version;
-    int REQUIRED_V = 1;
+    int REQUIRED_V = 2;
     store >> version;
     if(version!=REQUIRED_V) {
-        throw ML::Exception(ML::format(
+        throw HttpReturnException(400, ML::format(
```

Comment:
@nicolaskruchten can you weigh in on this please?  If we do care about backwards compatibility, do you have a stats table binary file generated with a previous version that we can use in a unit test?